src/main/scala/firrtl/ExecutionOptionsManager.scala

```diff
@@ -283,8 +284,8 @@ trait HasFirrtlOptions {
 
   parser.opt[String]("annotation-file")
     .abbr("faf")
-    .valueName ("<output>").
-    foreach { x =>
+    .valueName ("<input-anno-file>>")
```

Comment:
Is this intended to have an extra right bracket?

Reply:
nope, i'll fix it.